web/src/components/torrents/TorrentCreatorDialog.tsx

```diff
@@ -0,0 +1,366 @@
+      queryClient.invalidateQueries({ queryKey: ["torrent-creation-tasks", instanceId] })
+      queryClient.invalidateQueries({ queryKey: ["active-task-count", instanceId] })
+      toast.success("Torrent creation task queued")
+    },
+    onError: (err: Error) => {
+      setError(err.message)
+      toast.error(err.message || "Failed to create torrent task")
+    },
+  })
+
+  const form = useForm({
+    defaultValues: {
+      sourcePath: "",
+      private: false,
+      trackers: "",
+      comment: "",
+      source: "",
+      startSeeding: true,
+      // Advanced options
+      format: "" as TorrentFormat | "",
+      pieceSize: "",
+      torrentFilePath: "",
+      urlSeeds: "",
+    },
+    onSubmit: async ({ value }) => {
+      setError(null)
+
+      // Parse trackers (one per line)
+      const trackers = value.trackers
+        ?.split("\n")
+        .map((t) => t.trim())
+        .filter(Boolean)
+
+      // Parse URL seeds (one per line)
+      const urlSeeds = value.urlSeeds
+        ?.split("\n")
+        .map((u) => u.trim())
+        .filter(Boolean)
+
+      const params: TorrentCreationParams = {
+        sourcePath: value.sourcePath,
+        private: value.private,
+        trackers: trackers && trackers.length > 0 ? trackers : undefined,
+        comment: value.comment || undefined,
+        source: value.source || undefined,
+        startSeeding: value.startSeeding, // Always send boolean value
+        // Advanced options
+        format: value.format || undefined,
+        pieceSize: value.pieceSize ? parseInt(value.pieceSize) : undefined,
+        torrentFilePath: value.torrentFilePath || undefined,
+        urlSeeds: urlSeeds && urlSeeds.length > 0 ? urlSeeds : undefined,
+      }
+
+      mutation.mutate(params)
+    },
+  })
+
+  return (
+    <Dialog open={open} onOpenChange={onOpenChange}>
+      <DialogContent className="max-w-2xl max-h-[90vh] overflow-y-auto">
+        <DialogHeader>
+          <DialogTitle>Create Torrent</DialogTitle>
+          <DialogDescription>
+            Create a new .torrent file from a file or folder on the server
+          </DialogDescription>
+        </DialogHeader>
+
+        <form
+          onSubmit={(e) => {
+            e.preventDefault()
+            e.stopPropagation()
+            form.handleSubmit()
+          }}
+          className="space-y-4"
+        >
+          {error && (
+            <Alert variant="destructive">
+              <AlertCircle className="h-4 w-4" />
+              <AlertDescription>{error}</AlertDescription>
+            </Alert>
+          )}
+
+          {/* Source Path */}
+          <form.Field name="sourcePath">
+            {(field) => (
+              <div className="space-y-2">
+                <Label htmlFor="sourcePath">
+                  Source Path <span className="text-destructive">*</span>
+                </Label>
+                <Input
+                  id="sourcePath"
+                  placeholder="/path/to/file/or/folder"
```

Comment:
Since linux users get the love of a valid example, there should be some note for windows users about double backslash requirement.

If https://github.com/autobrr/go-qbittorrent/pull/84/files becomes a thing, it would be even nicer to just handle paths OS dependently.